### PR TITLE
Module 2 day 4 saddam dev

### PR DESCRIPTION
**Pull request by Saddamkhanlavani**
Added task1/2/3 files for day4-module-2.

### CODE REVIEW

File: Module-2/Day-4/Task/Saddam/Task_3.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            show:'Brooklyn 99'
+
+        }
+
+      ]
+      const name=[];
+      const company=[];
+      const designation=[];
+      const show=[];
+for(i=0; i<list.length; i++)
```
Comment (IshaaqhShaik) [inline Module-2/Day-4/Task/Saddam/Task_3.html:36]: correct the indentation here.

File: Module-2/Day-4/Task/Saddam/Task_2.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <script>
+        for (let i = 0; i < 4; i++) {
```
Comment (IshaaqhShaik) [inline Module-2/Day-4/Task/Saddam/Task_2.html:12]: for (let i = 0; i < 4; i++) {
            let str = " ";
            for (let j = 0; j <= 7; j++) {

                if (j >= i && j <= (7 - i)) {
                    str += "*";
                }
                else {
                    str += "_";
                }
            }
            console.log(str);
        }

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]

File: Module-2/Day-4/Task/Saddam/Task_3.html (hunk 1 above)
Comment (Saddamkhanlavani) [reply Module-2/Day-4/Task/Saddam/Task_3.html]: corrected bhieyya

File: Module-2/Day-4/Task/Saddam/Task_2.html (hunk 2 above)
Comment (Saddamkhanlavani) [reply Module-2/Day-4/Task/Saddam/Task_2.html:12]: modified bhieyya.

**Review by IshaaqhShaik — APPROVED** [review]

**Pull request merged**